Userland/Applications/FileManager/FileUtils.cpp

```diff
@@ -66,9 +35,7 @@ void delete_paths(Vector<String> const& paths, bool should_confirm, GUI::Window*
             return;
     }
 
-    for (auto& path : paths) {
-        delete_path(path, parent_window);
-    }
+    run_file_operation(FileOperation::Delete, paths, "", parent_window);
```

Comment:
Please use the null String (or even better would be `Optional<String>`, but this is not necessary here), so `{}` instead of `""`. This has a different semantic meaning, and the "" has a `StringImpl`, while the null String doesn't.

Reply:
Sure! I'll keep that in mind in the future.